packages/ui/react-ui-editor/src/themes/default.ts

```diff
@@ -49,12 +49,12 @@ export const defaultTheme: ThemeStyles = {
   '&.cm-focused': {
     outline: 'none',
   },
-  '& .cm-scroller': {
+  '.cm-scroller': {
```

Comment:
oh i thought this was a requirement to anchor a specific instance (e.g., if there are multiple instances on the page).